cmd/kwil-admin/cmds/setup/init.go

```diff
@@ -105,6 +106,25 @@ func initCmd() *cobra.Command {
 					}
 				}
 			}
+
+			// saves genesis state snapshot file in the root directory under the name "genesis_state.sql.gz"
+			if genesisState != "" {
+				if genesisState, err = common.ExpandPath(genesisState); err != nil {
+					return display.PrintErr(cmd, err)
+				}
+
+				file, err := os.ReadFile(genesisState)
+				if err != nil {
+					return display.PrintErr(cmd, err)
+				}
+
+				stateFile := filepath.Join(expandedDir, config.GenesisStateFileName)
+				err = os.WriteFile(stateFile, file, 0644)
+				if err != nil {
+					return display.PrintErr(cmd, err)
+				}
```

Comment:
We should also set the `kwild` command's genesis state config to point to the newly copied config

Reply:
genesis-state has now a default value set to `genesis-state.sql.gz`